File: tests/machine/x/typescript/python_math.ts
const math = { pi: Math.PI, e: Math.E, sqrt: Math.sqrt, pow: Math.pow, sin: Math.sin, log: Math.log };

const r = 3;
const area = (math.pi * math.pow(r, 2));
const root = math.sqrt(49);
const sin45 = math.sin((math.pi / 4));
const log_e = math.log(math.e);
console.log("Circle area with r =", r, "=>", area);
console.log("Square root of 49:", root);
console.log("sin(π/4):", sin45);
console.log("log(e):", log_e);

File: tests/machine/x/cs/python_math.cs
using System;

static class math
{
    public const double pi = Math.PI;
    public const double e = Math.E;
    public static double sqrt(double x) { return Math.Sqrt(x); }
    public static double pow(double x, double y) { return Math.Pow(x, y); }
    public static double sin(double x) { return Math.Sin(x); }
    public static double log(double x) { return Math.Log(x); }
}

class Program
{
    static void Main()
    {
        double r = 3.000000;
        double area = (math.pi * math.pow(r, 2.000000));
        double root = math.sqrt(49.000000);
        double sin45 = math.sin((math.pi / 4.000000));
        double log_e = math.log(math.e);
        Console.WriteLine(string.Join(" ", new[] { Convert.ToString("Circle area with r ="), Convert.ToString(r), Convert.ToString("=>"), Convert.ToString(area) }));
        Console.WriteLine(string.Join(" ", new[] { Convert.ToString("Square root of 49:"), Convert.ToString(root) }));
        Console.WriteLine(string.Join(" ", new[] { Convert.ToString("sin(π/4):"), Convert.ToString(sin45) }));
        Console.WriteLine(string.Join(" ", new[] { Convert.ToString("log(e):"), Convert.ToString(log_e) }));
    }
}
